docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -3,88 +3,87 @@ title: Visual Studio Tools for Docker on Windows
+When you want to compose a multi-container solution, add container orchestration support to your projects. This lets you run and debug a group of containers (a whole solution) at the same time if they're defined in the same *docker-compose.yml* file.
 
-To add container orchestration support, right-click on the solution or project node in **Solution Explorer**, and choose **Add** > **Container Orchestration Support**. Then choose **Docker Compose** or **Service Fabric** to manage the containers.
+To add container orchestration support, right-click on the solution or project node in **Solution Explorer**, and choose **Add > Container Orchestration Support**. Then choose **Docker Compose** or **Service Fabric** to manage the containers.
 
-After you add container orchestration support to your project, you see a Dockerfile added to the project and a **docker-compose** folder added to the solution in **Solution Explorer**, as shown in Figure 4-29:
+After you add container orchestration support to your project, you see a Dockerfile added to the project and a **docker-compose** folder added to the solution in **Solution Explorer**, as shown in Figure 4-34:
 
 ![Docker files in Solution Explorer in Visual Studio](media/docker-support-solution-explorer.png)
 
-Figure 4-29: Docker files in Solution Explorer in Visual Studio 2017
+**Figure 4-34**. Docker files in Solution Explorer in Visual Studio 2017
 
 If *docker-compose.yml* already exists, Visual Studio just adds the required lines of configuration code to it.
 
 ## Configure Docker tools
 
-From the main menu, choose **Tools** > **Options**, and expand **Container Tools** > **Settings**. The container tools settings appear.
+From the main menu, choose **Tools > Options**, and expand **Container Tools > Settings**. The container tools settings appear.
 
-![](./media/visual-studio-docker-tools-options.png)
+![Visual Studio Docker tools options, showing: Automatically pull required Docker images on project load, Automatically start containers in background, Automatically kill containers on solution close, and Do not prompt for trusting SSL certificate.](./media/visual-studio-docker-tools-options.png)
 
-Figure 4-30: Docker Tools Options
+**Figure 4-35**. Docker Tools Options
 
 The following table might help you decide how to set these options.
 
 | Name | Default Setting | Applies To | Description |
 | -----|:---------------:|:----------:| ----------- |
 | Automatically pull required Docker images on project load | On | Docker Compose | For increased performance, when loading projects, Visual Studio will start a Docker pull operation in the background so that when you are ready to run your code, the image is already downloaded or in the process of downloading. If you're just loading projects and browsing code, you can turn this off to avoid downloading container images you don't need. |
```

Comment:
```suggestion
| Automatically pull required Docker images on project load | On | Docker Compose | For increased performance when loading projects, Visual Studio will start a Docker pull operation in the background so that when you are ready to run your code, the image is already downloaded or in the process of downloading. If you're just loading projects and browsing code, you can turn this off to avoid downloading container images you don't need. |
```